toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -17,12 +17,12 @@ bzip2-1.0.6-15.cm1.x86_64.rpm
 bzip2-debuginfo-1.0.6-15.cm1.x86_64.rpm
 bzip2-devel-1.0.6-15.cm1.x86_64.rpm
 bzip2-libs-1.0.6-15.cm1.x86_64.rpm
-ca-certificates-20200720-9.cm1.noarch.rpm
-ca-certificates-base-20200720-9.cm1.noarch.rpm
-ca-certificates-legacy-20200720-9.cm1.noarch.rpm
-ca-certificates-microsoft-20200720-9.cm1.noarch.rpm
-ca-certificates-shared-20200720-9.cm1.noarch.rpm
-ca-certificates-tools-20200720-9.cm1.noarch.rpm
+ca-certificates-20200720-10.cm1.noarch.rpm
+ca-certificates-base-20200720-10.cm1.noarch.rpm
+ca-certificates-legacy-20200720-10.cm1.noarch.rpm
+ca-certificates-microsoft-20200720-10.cm1.noarch.rpm
+ca-certificates-shared-20200720-10.cm1.noarch.rpm
+ca-certificates-tools-20200720-10.cm1.noarch.rpm
```

Comment:
Add `ca-certificates-static` to this list (and the aarch64 manifest)

Reply:
Good catch!